addons/sale/models/res_company.py

```diff
@@ -34,9 +34,12 @@ def action_close_sale_quotation_onboarding(self):
     @api.model
     def action_open_sale_onboarding_payment_acquirer(self):
         """ Called by onboarding panel above the quotation list."""
-        self.env.company.get_chart_of_accounts_or_fail()
-        action = self.env["ir.actions.actions"]._for_xml_id("sale.action_open_sale_onboarding_payment_acquirer_wizard")
-        return action
+        return self._start_onboarding_payment_acquirer('sale')
+
+    def _set_payment_acquirer_onboarding_step_done(self, onboarding):
+        super()._set_payment_acquirer_onboarding_step_done(onboarding)
+        if onboarding == 'sale':
+            self.env.company.sudo().set_onboarding_step_done('sale_onboarding_order_confirmation_state')
```

Comment:
@AntoineVDV @antonylesuisse 
This modification induces a sale_tour failure, see https://runbot.odoo.com/runbot/build/11732352 . (Since the proxy is not up yet, it won't jump to Stripe)

Since we will jump outsite of the odoo client, there are no way to fix the test.

Reply:
Context : From the Sale Onboarding, we will try to directly start the Stripe Onboarding rather than giving the choice to the user to select "Electronic Signature".

Reply:
Then "tant pis", we don't test this step.